.ci/test_nnlibcuda.jl

```diff
@@ -0,0 +1,9 @@
+import Pkg
+# import NNlibCUDA
+pkgs = ["NNlibCUDA"]
+
+# Do this for the time being since test doesn't pick up the manifest
+# for some reason. Can remove this and manifests when cuda 3.0 is released. 
+Pkg.add(url="https://github.com/JuliaGPU/CUDA.jl.git", rev="master")
```

Comment:
@maleadt I'm trying to point to CUDA master but I'm having no luck. This line is working for me executing the script locally but doesn't work with buildkite. Do you have any suggestions?

Reply:
Also the Manifest seems to be ignored when executing this script

Reply:
It doesn't get to that but fails during precompilation, after installing the package. The only solution is to add a Manifest, I guess.

Reply:
I tried locally but `test` didn't see the Manifest, don't know why. uploaded it here, let's see

Reply:
CUDA.jl also has the manifest committed, and that works.

Reply:
I don't know why it doesn't work here, anyways I worked around moving the Pkg.add before the precompile. 
Can be revisited later